src/index.js

```diff
@@ -281,6 +282,40 @@ module.exports = function ChannelsMiddleware(mwOpts) {
 							};
 						}
 
+						// Attach method to simplify unit testing
+						if (!svc[mwOpts.channelHandlerTrigger]) {
+							/**
+							 * Call a local channel event handler. Useful for unit tests.
+							 *
+							 * @param {String} channelName
+							 * @param {Object} payload
+							 * @param {Object} rawMessage
+							 * @returns
+							 */
+							svc[mwOpts.channelHandlerTrigger] = (channelName, payload, raw) => {
+								if (!svc.schema[mwOpts.schemaProperty][channelName])
+									return Promise.reject(
+										new MoleculerError(
+											`'${channelName}' is not registered as local channel event handler`,
+											500,
+											"NOT_FOUND_CHANNEL",
+											{ channelName }
+										)
+									);
+
```

Comment:
Please add a `svc.logger.debug` that the handler triggered via "emitLocalChannelHandler". It could be useful, if somebody faces suspicious calls because the handler triggered, but no data via adapter and turn on debug logs he will see the handler is called via this method and not via adapter.

Reply:
Something like this?
```js
svc.logger.debug(
   `${mwOpts.channelHandlerTrigger} called '${channelName}' handler`
);
```

Reply:
Yeah.

Reply:
Done